src/plugin/declarativedbusinterface.cpp

```diff
@@ -90,8 +90,8 @@
     \section2 Calling D-Bus Methods
 
     Remote D-Bus methods can be called using either \l call() or \l typedCall(). \l call() provides
-    a simplier calling API, only supporting basic data types and discards any value return by the
-    method. \l typedCall() supports more data types and has callbacks for call completion and error.
+    a simplier calling API, only supporting basic data types, \l typedCall()
```

Comment:
Here being the existing 'simplier', if you don't mind fixing while at it.

Also if you don't mind, could still rebase on top of current master to have slightly cleaner git history.